.mailmap

```diff
@@ -54,6 +54,7 @@ formalize.eth <formalize@protonmail.com>           ilya <ilya@localhost.localdom
 Andres Erbsen <andreser@mit.edu>                   Andres Erbsen <andres@kevix.co>
 Jim Fehrle <jfehrle@sbcglobal.net>                 Jim <jfehrle@sbcglobal.net>
 Jim Fehrle <jfehrle@sbcglobal.net>                 Jim Fehrle <jim.fehrle@gmail.com>
+Jim Fehrle <jfehrle@sbcglobal.net>                 jfehrle <jfehrle@users.noreply.github.com>
```

Comment:
What's the purpose of this list?  Coq-related correspondence should go to my Gmail account.

Reply:
This list is intended to deduplicate the contributors' list from the git logs. Not used to send any e-mail. But we can put the Gmail account as the primary thing if you wish.

Reply:
That would be good, thanks.

Reply:
Done